solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/util/XMLMessageMapper.java

```diff
@@ -225,7 +228,18 @@ public XMLMessage map(Message<?> message, Collection<String> excludedHeaders, bo
 			if (!header.getValue().isReadable()) {
 				continue;
 			}
-			builder.setHeaderIfAbsent(header.getKey(), header.getValue().getReadAction().apply(xmlMessage));
+			if (ignoredHeaderProperties.contains(header.getKey())) {
+				continue;
+			}
+			try {
+				builder.setHeaderIfAbsent(header.getKey(), header.getValue().getReadAction().apply(xmlMessage));
+			} catch (UnsupportedOperationException e) {
+				if (logger.isDebugEnabled()) {
+					logger.debug(String.format("Ignoring Solace header %s. Error: %s", header.getKey(), e));
```

Comment:
Could you either make this debug log print the whole stacktrace or add an alternative path if `logger.isTraceEnabled()` is `true` which does? Because if we, a user, or support needs to needs to debug the origin of this `UnsupportedOperationException`, it would currently not be possible if the log only prints the top-level error message.